packages/cli/src/utils/platform.ts

```diff
@@ -59,6 +59,21 @@ export const getRequiredParameter = <$Args extends Record<string, unknown>, $Nam
   return value
 }
 
+export const ErrorPlatformUnauthorized = new Error(
+  'No platform credentials found. Please provide a token via --token or -t, or add PRISMA_TOKEN environment variable or run `prisma platform login`.',
+)
+
+export const getPlatformToken = async <$Args extends Record<string, unknown>>(args: $Args) => {
+  try {
+    const token =
+      getOptionalParameter(args, ['--token', '-t'], 'PRISMA_TOKEN') ||
+      (await readAuthConfig().catch(() => ({ token: '' }))).token
+    return token
+  } catch (error) {
+    return ``
+  }
+}
+
```

Comment:
what do you think on making this one `getPlatformTokenOrThrow` ? seems like consumers are also throwing this error 
I think this can remove some code repetition across the different commands

Reply:
I agree with you. Initially I actually had this function throwing.

@Jolg42 are there any code guidelines/paradigms about returning an error vs throwing an error?

In this specific context, it would make sense to throw, avoiding code repetition as @tatianainama pointed out.

Reply:
I think you'll be ok with throwing here since you know where this is used exactly. (at the beginning of a command).

Reply:
@tatianainama I converted this utility to throw! Anything else to change or good to go?